src/vs/editor/contrib/inlineCompletions/ghostTextController.ts

```diff
@@ -173,6 +187,50 @@ registerEditorCommand(new GhostTextCommand({
 	}
 }));
 
+export class ShowNextInlineCompletionAction extends EditorAction {
+	constructor() {
+		super({
+			id: 'editor.action.showNextInlineCompletion',
+			label: nls.localize('showNextInlineCompletion', "Show Next Inline Completion"),
+			alias: 'Show Next Inline Completion',
+			precondition: EditorContextKeys.writable,
+			kbOpts: {
+				weight: 100,
+				primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyMod.Shift | KeyCode.RightArrow,
+			},
+		});
+	}
+
+	public async run(accessor: ServicesAccessor | undefined, editor: ICodeEditor): Promise<void> {
+		const controller = GhostTextController.get(editor);
+		if (controller) {
+			controller.showNextInlineCompletion();
+		}
+	}
+}
+
+export class ShowPreviousInlineCompletionAction extends EditorAction {
+	constructor() {
+		super({
+			id: 'editor.action.showPreviousInlineCompletion',
+			label: nls.localize('showPreviousInlineCompletion', "Show Previous Inline Completion"),
+			alias: 'Show Previous Inline Completion',
+			precondition: EditorContextKeys.writable,
+			kbOpts: {
+				weight: 100,
+				primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyMod.Shift | KeyCode.LeftArrow,
+			},
+		});
+	}
+
+	public async run(accessor: ServicesAccessor | undefined, editor: ICodeEditor): Promise<void> {
+		const controller = GhostTextController.get(editor);
+		if (controller) {
+			controller.showNextInlineCompletion();
```

Comment:
@hediet, this should be `controller. showPreviousInlineCompletion();`, shouldn't it?

Reply:
Already fixed (will be available tomorrow). In my test I only had two suggestions, so previous and next did the same.